ChaosMod/Memory/ViewMatrix.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include "Memory.h"
+#include "Handle.h"
+#include "Entity.h"
+
+#include "../Util/Natives.h"
```

Comment:
```suggestion
#include "Util/Natives.h"
```
Both work but we use this style